modules/photo/test/test_hdr.cpp

```diff
@@ -227,7 +227,7 @@ TEST(Photo_CalibrateDebevec, regression)
     diff = diff.mul(1.0f / response);
     double max;
     minMaxLoc(diff, NULL, &max);
-    ASSERT_FALSE(max > 0.1);
+    ASSERT_LT(max, 0.131);
```

Comment:
Please use this guard for ARM tolerance values: `#if defined(__arm__) || defined(__aarch64__)`